tokio/src/process/mod.rs

```diff
@@ -952,6 +954,16 @@ impl Child {
         }
     }
 
+    /// Extracts the raw handle of the process associated with this child while
+    /// it is still running.
```

Comment:
```suggestion
    /// it is still running. Returns `None` if the child has exited.
```

Minor documentation clarification